rider-fsharp/testData/markup/FSharpHoverDocTest/test hover docs for EntryPoint/gold/Program.gold

```diff
@@ -1,3 +1,3 @@
-<div style="padding-bottom:5px;margin-bottom:5px;border-bottom: thin solid #d1d1d1;"><span style="color:#000080"><div class='definition'><pre style='word-wrap: break-word; white-space: pre-wrap;'></span><span style="color:#000080">new</span> : <span style="color:#000000">unit</span> -&gt; <span style="color:#000000">EntryPointAttribute</pre></div></span><div class='content'>Creates an instance of the attribute</div><table class='sections'><tr><td valign='top' class='section'><p>Returns:</td><td valign='top'><p>EntryPointAttribute</td></table></div><span style="color:#000080"><div class='definition'><pre style='word-wrap: break-word; white-space: pre-wrap;'></span><span style="color:#000080">type</span> <span style="color:#000000">EntryPointAttribute</span> <span style="color:#000000">=</span><br>  <span style="color:#000080">inherit</span> <span style="color:#000000">Attribute</span><br>  <span style="color:#000080">new</span> : <span style="color:#000000">unit</span> -&gt; <span style="color:#000000">EntryPointAttribute</pre></div></span><div class='content'>Adding this attribute to a function indicates it is the entrypoint for an application. If this attribute is not specified for an EXE then the initialization implicit in the module bindings in the last file in the compilation sequence are used as the entrypoint.</div>
+<div style="padding-bottom:5px;margin-bottom:5px;border-bottom: thin solid #d1d1d1;"><span style="color:#000080"><div class='definition'><pre style='word-wrap: break-word; white-space: pre-wrap;'></span><span style="color:#000080">new</span> : <span style="color:#000000">unit</span> -&gt; <span style="color:#000000">EntryPointAttribute</pre></div></span><div class='content'>Creates an instance of the attribute</div><table class='sections'><tr><td valign='top' class='section'><p>Returns:</td><td valign='top'><p>EntryPointAttribute</td></table></div><span style="color:#000080"><div class='definition'><pre style='word-wrap: break-word; white-space: pre-wrap;'></span><span style="color:#000080">type</span> <span style="color:#000000">EntryPointAttribute</span> <span style="color:#000000">=</span><br>  <span style="color:#000080">inherit</span> <span style="color:#000000">Attribute</span><br>  <span style="color:#000080">new</span> : <span style="color:#000000">unit</span> -&gt; <span style="color:#000000">EntryPointAttribute</pre></div></span><div class='content'>Adding this attribute to a function indicates it is the entrypoint for an application. If this attribute is not specified for an EXE then the initialization implicit in the module bindings in the last file in the compilation sequence are used as the entrypoint.</div>
```

Comment:
@DedSec256 @kirillgla Is there a way to make these dumps easier to read? It's quite hard to spot the difference here.

Reply:
Not really, no. I use diffchecker to see the difference in cases like this one